text/chapter11.md

```diff
@@ -623,20 +625,21 @@ class (Applicative f, Plus f) <= Alternative f
 
 `Alternative` provides two new combinators: the `empty` value, which provides a prototype for a failing computation, and the `alt` function (and its alias, `<|>`) which provides the ability to fall back to an _alternative_ computation in the case of an error.
 
-The `Data.List` module provides two useful functions for working with type constructors in the `Alternative` type class:
+The `Data.Array` module provides two useful functions for working with type constructors in the `Alternative` type class:
 
 ```haskell
-many :: forall f a. Alternative f => Lazy (f (List a)) => f a -> f (List a)
-some :: forall f a. Alternative f => Lazy (f (List a)) => f a -> f (List a)
+many :: forall f a. Alternative f => Lazy (f (Array a)) => f a -> f (Array a)
+some :: forall f a. Alternative f => Lazy (f (Array a)) => f a -> f (Array a)
 ```
 
+There is also an equivalent `many` and `some` for `Data.List`
+
 The `many` combinator uses the `Alternative` type class to repeatedly run a computation _zero-or-more_ times. The `some` combinator is similar, but requires at least the first computation to succeed.
 
 In the case of our `Parser` monad transformer stack, there is an instance of `Alternative` induced by the `ExceptT` component, which supports failure by composing errors in different branches using a `Monoid` instance (this is why we chose `Array String` for our `Errors` type). This means that we can use the `many` and `some` functions to run a parser multiple times:
 
 ```text
-> import Split
```

Comment:
The book snippets contains all the code for writing your own `split` function, so I think users will likely just continue to use their own version, rather than importing the pre-written version. At least that's what happened when I read through. Considering removing `Split.purs`.